README.md

```diff
@@ -319,8 +319,7 @@ the dcrdata workload.
 Read [postgresql-tuning.conf](./db/dcrpg/postgresql-tuning.conf) carefully for
 details on how to make the necessary changes to your system. A helpful online
 tool for determining good settings for your system is called
-[PGTune](https://pgtune.leopard.in.ua/). Note that when using this tool to
-subtract 1.5-2GB from your system RAM so dcrdata itself will have plenty of
+[PGTune](https://pgtune.leopard.in.ua/). Note that when using this tool, subtract 1.5-2GB from your system RAM so dcrdata itself will have plenty of
```

Comment:
wrapping looked right before

Reply:
Maybe, wasn't fixing the wrapping though, that sentence didn't read well.

Reply:
Do you think we could use `markdownlint` on dcrdata?